front/public/locales/en/operationalStudies/scenario.json

```
@@ -57,7 +57,7 @@
       "simulation_failed": "Simulation failed",
       "rolling_stock_not_found": "RS not found"
     },
-    "invalidTrains": "The timetable contains invalid trains.",
+    "invalidTrains": "Some trains are invalid.",
```

No point in mockup (same for fr)

```suggestion
    "invalidTrains": "Some trains are invalid",
```

---

Done